app/experimenter/kinto/tests/test_tasks.py

```diff
@@ -32,6 +33,10 @@ def test_push_experiment_to_kinto_sends_experiment_data(self):
             if_not_exists=True,
         )
 
+        self.assertTrue(
+            ExperimentBucketRange.objects.filter(experiment=self.experiment).exists()
+        )
+
```

Comment:
I think here is where we'd want to check that the buckets are generated using hte preset values.